### PR TITLE
[ENH, MRG] Add info save method

### PR DESCRIPTION
**Pull request by alexrockhill**
Adds a save method to the info class in addition to `mne.io.write_info` for convenience.

### CODE REVIEW

File: mne/io/meas_info.py [hunk 1]
```diff
@@ -1227,6 +1227,22 @@ def _repr_html_(self, caption=None):
...
+        Parameters
+        ----------
+        fname : str
+            The name of the file. Should end by -info.fif.
+        data_type : int
+            The data_type in case it is necessary. Should be 4 (FIFFT_FLOAT),
+            5 (FIFFT_DOUBLE), or 16 (FIFFT_DAU_PACK16) for
+            raw data.
+        reset_range : bool
+            If True, info['chs'][k]['range'] will be set to unity.
```
Comment (larsoner) [inline mne/io/meas_info.py:1242]: Can we avoid exposing these? I'd rather add them later if someone needs them

Comment (alexrockhill) [reply mne/io/meas_info.py:1242]: This is copied from `mne.io.write_info`, do you want to remove it there too?

Comment (larsoner) [reply mne/io/meas_info.py:1242]: No we need it there for completeness. Stuff in mne.io.* was added quite early and not with a user API in mind. This PR is a chance for us to make a nice API from the start

Comment (alexrockhill) [reply mne/io/meas_info.py]: Sounds good

File: mne/io/meas_info.py [hunk 2]
```diff
@@ -1227,6 +1227,16 @@ def _repr_html_(self, caption=None):
             experimenter=self.get('experimenter'),
         )
 
+    def save(self, fname):
+        """Write measurement info in fif file.
+
+        Parameters
+        ----------
+        fname : str
+            The name of the file. Should end by -info.fif.
```
Comment (hoechenberger) [inline mne/io/meas_info.py:1236]:
```suggestion
            The name of the file. Should end by `-info.fif`.
```

Comment (larsoner) [reply mne/io/meas_info.py]: Edited your comment to have double backticks (rst vs md code mode)

**Review by hoechenberger — APPROVED** [review]

**Review by agramfort — APPROVED** [review]

**Comment by drammock** [comment]
thanks @alexrockhill

**Pull request merged**